### ISSUE TITLE
Option to exclude nested functions/values from Structure View

### ISSUE THREAD

**Issue by Herteby** [klazuka/intellij-elm#562]
This new feature may be great for some, but personally I find that it adds too much noise. It would be nice to have a checkbox somewhere to toggle it :slightly_smiling_face:

**Comment by klazuka** [comment]
Agreed. I use the structure view popup for quick, top-level navigation (cmd-F12 on macOS), and I think it works best when it's flat.

@Herteby are you referring to the permanent panel on the left side of the screen or the popup?

**Comment by Herteby** [comment]
I use both the permanent panel and the popup. I usually open the popup and then type to search by name, and in that case I would like to only find top-level definitions. For the side panel it's less of a big deal.
